python/py-soco/Portfile

```diff
@@ -35,9 +35,15 @@ if {${name} ne ${subport}} {
 
     depends_test-append \
                     port:py${python.version}-pip \
+                    port:py${python.version}-pyflakes \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-requests-mock \
```

Comment:
I kind of doubt that `pip` and `wheel` are needed as test dependencies - can you try removing them and running the tests in trace-mode to make sure they're actually required?

Reply:
@reneeotten before I've switched to pytest it required. Perhabs by some side effects. Anyway, until 0.25.1 tests were broken.